samples/Config.Providers.Web/project.json

```diff
@@ -16,5 +16,8 @@
   },
   "commands": {
```

Comment:
I could but I'm under the impressions that most people are still using dnx/dnu and it's probably best to preserve the ability to run these samples for now until most people have switched to dnx/dnu? For one, I'm not sure if VS tooling has caught up to using dotnet to run samples.